index.src.html

```diff
@@ -193,17 +193,26 @@ <h2 id="terms">Key Concepts and Terminology</h2>
       with that <a>environment settings object</a> as their <a for=request>client</a>.
     </dd>
 
-    <dt><dfn>same-origin request</dfn></dt>
+    <dt><dfn>resolved referrer</dfn></dt>
     <dd>
-      A {{Request}} <var>request</var> is a <strong>same-origin request</strong>
-      if <var>request</var>'s <a for=request>origin</a> and the <a for=url>origin</a> of
-      <var>request</var>'s <a for=request>current url</a> are <a lt="same origin">the same</a>.
+      A {{Request}} <var>request</var>'s <strong>resolved referrer</strong> is a <a for=/>URL</a>
```

Comment:
This term is a bit unintuitive to me; I would have never guessed what it meant before reading the definition. What do you think about "referrer source" or "referrer source URL"?

Reply:
Thanks for the feedback, I think this is resolved given the comment on the below conversation.